README.md

```diff
@@ -56,6 +56,12 @@ The `configType` variable will be either `"DEVELOPMENT"` or `"PRODUCTION"`.
 
 The function should return the updated Vite configuration.
 
+## Note about working directory
+
+The builder will by default enable Vite's [server.fsServe.strict](https://vitejs.dev/config/#server-fsserve-strict)
+option, for increased security. The default `server.fsServe.root` is set to the parent directory of the
```

Comment:
It looks like `root` is being set at the root of the config, not nested under `server.fsServe`, isn't it?  Shouldn't they be changing their top-level root if we guess it wrong?

Reply:
You're right! There are two `root` config options (one top-level and one under server.fsServe) and they behave a bit differently. I updated the PR.